docs/design/w3c/spec.md

```diff
@@ -0,0 +1,545 @@
+include `credentialSchema` property to enforce a specific structure on a given verifiable credential and encoding used
+to map the claims of a verifiable credential to an alternative representation format.
+
+In the context of W3C AnonCreds credentials defined a custom `AnonCredsDefinition` data schema in order to include the
+following information to credential:
+
+In the case of W3C AnonCreds credentials, the `credentialSchema` attribute defines a custom `AnonCredsDefinition`
+schema in order to include the information about Indy related definitions to credential and looks the following:
+
+```
+{
+  ... 
+  "credentialSchema": {
+    "type": "AnonCredsDefinition",
+    "definition": "did:sov:3avoBCqDMFHFaKUHug9s8W:3:CL:13:default",
+    "schema": "did:sov:3avoBCqDMFHFaKUHug9s8W:2:fabername:0.1.0",
+    "encoding": "auto"
+  },
+  ...
+}
+```
+
+* `schema` - id of Indy Schema
+* `definition` - id of Indy Credential Definition
+* `revocation` - Revocation Registry Accumulator value
+* `witness` - Witness value
+* `encoding` - attributes encoding algorithm
+    * encoded credential attribute values (binary representation required for doing CL signatures) are not included
+      neither to `credentialSubject` or `signature`
+    * `encoding: auto` implies using the algorithm defined
+      at [Aries RFC 0592 Indy Attachments section](https://github.com/hyperledger/aries-rfcs/tree/main/features/0592-indy-attachments#encoding-claims)
+      to generate encoded values under the hood during the signature generation and proof verification.
+
+#### Issuer
+
+W3C [Issuer](https://www.w3.org/TR/vc-data-model/#issuer) section requires including of `issuer` property to express the
+issuer of a verifiable credential.
+
+In the case of W3C AnonCreds credentials, the `issuer` attribute should be represented as a
+resolvable [DID URL](https://w3c-ccg.github.io/did-resolution/) and looks the following:
+
+```
+{
+  ... 
+  "issuer": "did:sov:3avoBCqDMFHFaKUHug9s8W",
+  ...
+}
+```
+
+#### Issuance Date
+
+W3C [Issuance Date](https://www.w3.org/TR/vc-data-model/#issuance-date) section requires including of `issuanceDate`
+property to express the date and time when a credential becomes valid.
+
+In the case of W3C AnonCreds credentials, for the `issuanceDate` attribute recommended setting of a random time of the
+day when credential was issued or transformed and looks the following:
+
+```
+{
+  ... 
+  "issuanceDate": "2010-01-01T19:23:24Z",
+  ...
+}
+```
+
+#### Proofs (Signatures)
+
+W3C [Proofs (Signatures)](https://www.w3.org/TR/vc-data-model/#proofs-signatures) section requires including of `proof`
+property to express confirmation of the credential's validity.
+
+According to the specification, one or many proof objects can be added to verifiable credentials.
+In the context of W3C AnonCreds credentials included at least two proof object entries: AnonCreds CL
```

Comment:
Wording has to be handled carefully here. The plan outlined here specifies an AnonCreds signature as a Data Integrity Proof.  A second, non-AnonCreds signature would also be a Data Integrity Proof — just one using a different signature scheme.  So the wording here is not quite right.